js-w3-hw.js

```diff
@@ -0,0 +1,144 @@
+// EX1. Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+
+const nameToRemove = "Ahmad";
+const index = names.indexOf(nameToRemove); // Index of the name to remove
+
+if (index !== -1) {
+  // Check if the name is found, and then remove it
+  names.splice(index, 1);
+}
+
+console.log(names); // Log out the remaining array
+
+// EX2. When will we be there??
+
+function calculateTravelTime(travelInformation) {
+  const { speed, destinationDistance } = travelInformation;
+
+  // Calculate the time in hours as a decimal
+  const timeInHours = destinationDistance / speed;
+
+  // Extract the whole number of hours
+  const hours = Math.floor(timeInHours);
+
+  // Calculate the remaining minutes and round it to the nearest integer.
+  const minutes = Math.round((timeInHours - hours) * 60);
+
+  // Return the formatted result
+  return `${hours} hours and ${minutes} minutes`;
+}
+
+// Example
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
```

Comment:
Generally it's recommended to define global variables in the top of the files. It's tricky in this case when you have multiple functions in the same file, so best would be to either write a exercise per file, or to add the global variables as first thing in each exercise.